### PR TITLE
add the possibility to display vector field

### PR DESCRIPTION
**Pull request by kerautret**

### CODE REVIEW

File: visualisation/meshViewer.cpp [hunk 1]
```diff
@@ -76,45 +84,67 @@ int main( int argc, char** argv )
...
+  
+  
+  bool displayVectorField = vm.count("displayVectorField");
+  std::vector<unsigned int> vectFieldIndexes = {0,1,2,3,4,5};
+  
+  if (displayVectorField) {
+    if(vm.count("vectorFieldIndex")){
+      vectFieldIndexes = vm["vectorFieldIndex"].as<std::vector<unsigned int> >();
+      if (vectFieldIndexes.size() != 6) {
+        trace.warning() << "you should specify indexes for each of the 6 field of the two coordinates." << std::endl;
```
Comment (dcoeurjo) [inline visualisation/meshViewer.cpp]: indexes -> indices ?

**Comment by dcoeurjo** [comment]
ok for me (just typo). ;)

File: visualisation/meshViewer.cpp (hunk 1 above)
Comment (kerautret) [inline visualisation/meshViewer.cpp]: yes more common thanks ;)

**Comment by kerautret** [comment]
thanks for the remarks merging ;)

**Pull request merged**